packages/framework/src/Model/Customer/User/CustomerUserRepository.php

```diff
@@ -156,7 +164,7 @@ public function getCustomerListQueryBuilderByQuickSearchData(
     public function replaceUsersPricingGroup(PricingGroup $oldPricingGroup, PricingGroup $newPricingGroup)
```

Comment:
I would rename it to `replaceCustomerUsersPricingGroup`, also repository method too